src/index.ts

```diff
@@ -1,9 +1,38 @@
 import * as https from 'https'
-import * as tls from 'tls'
-import * as forge from 'node-forge'
 import * as mysql from 'mysql2'
 import type { Connection } from 'mysql2'
 import type { IncomingMessage } from 'http'
+import { customAlphabet } from 'nanoid'
+import * as crypto from 'crypto'
+import { Crypto as webCryptoPolyfill } from '@peculiar/webcrypto'
+import * as x509 from '@peculiar/x509'
+
+// Define interface for webcrypto. This is not available (yet)
+// in TypeScript for the native webcrypto so we need this to
+// satisfy the rest of the type checks.
+//
+// This definition is the same as what @peculiar/webcrypto uses
+// which functions as a polyfill for Node versions earlier than Node 15
+// which don't provide webcrypto.
+interface WebCrypto {
+  readonly subtle: SubtleCrypto
+  getRandomValues<T extends ArrayBufferView | null>(array: T): T
+}
+
+let webcrypto = crypto.webcrypto as unknown as WebCrypto
+// Need to cast here first to unknown and then to WebCrypto since
+// TypeScript doesn't know about WebCrypto as a specific type.
+if (crypto.webcrypto == undefined) {
+  console.warn(
+    'No native webcrypto available, using @peculiar/webcrypto polyfill. Please upgrade to Node 15 or later to avoid this warning.'
```

Comment:
@dgraham Do you think this warning is a good or bad idea here?

Reply:
Built-in crypto is a good reason to upgrade, so this looks great to me.